### DISCUSSION TITLE
exposing additional ports in kafka connect CRDs?

### DISCUSSION THREAD

**Discussion by cstmgl** [strimzi/strimzi-kafka-operator#7890] [Q&A] [answered]
Hi all, 
I'm trying to do some testing in my application/connect and I wanted to see if I could somehow expose additional ports on the generated Deployment YML.
I could patch almost everything with the connectContainer but I could not find a way to add additional ports.

I kind of expected it but I'm just wondering if I missed something or is there any other way I can do this besides stopping the strimzi cluster operator and edit the kafkaconnect deployment.yml generated by it the operator?

Basically I'm trying to do something like this:
https://www.yourkit.com/docs/java/help/docker.jsp

I did include that to my base image and then I would need to include something like:
                  "containerPort": port,
                  "protocol": "TCP",
                  "name": "profiler"

Thanks for any feedback and best regards,

**Comment by scholzj** [comment, marked as answer]
You cannot expose any custom ports. That is not supported by Strimzi.
